Tests/Compare-ADObjects.ps1

```diff
@@ -1,32 +1,32 @@
 # Source: https://learn.microsoft.com/en-us/archive/blogs/janesays/compare-all-properties-of-two-objects-in-windows-powershell
 param(
-    [Parameter(Mandatory=$true)]
+    [Parameter(Mandatory = $true)]
```

Comment:
Let's replace these with `[Parameter(Mandatory)]` for consistency with rest of codebase.

Reply:
Yes, I'm down with leaving off the redundant positive Boolean in parameter definitions.